Add needsUpgrade

DG12 opened the pull request:
If URL includes A# tell them there is an upgrade available.
Perhaps the suffix was missing a minute ago.
svg/fallback/now-available.png now included.

DG12 committed:
add upgradeNeeded

DG12 committed:
add needsUpgrade

If Beacon URL contains #A suggest the user tap on link to do firmware upgrade,
include a cookie that expires weekly so as not to be a pest if we told them recently

DG12 committed:
Darn, Reviewing changes shows little oops

correct capital B in Beacon and tag in now available link

DG12 commented:
I also added the svg/fallback/available.png file which was missing but I don't see it included in this Pull Request!

DG12 committed:
now=available.png was missing

ojousima commented on css/ruu-vi-styles.css:
```diff
@@ -212,7 +218,7 @@ p {
   line-height: 24px;
   line-height: 2.4rem;
   letter-spacing: 2px;
-  text-transform: uppercase;
```
Let's keep the uppercase

ojousima commented:
Is the site available somewhere online for review?

DG12 commented:
http://mybeacons.info/index.1.1.3.html#AIgbAMLN

You might want to change 
To css/ruu-vi-styles.css .needsUpgrade  {font-family:sans-serif**;text-transform: none;**}
so entire sentence is no upper case.

DG12 commented:
By the way what do you use to edit html?

DG12 committed:
revise transform:uppercase/none

restore transform:uppercase to <P> (I had thought Otso had reverted it by accident)
include transform:none to needsUpgrade.
PS sorry for the delay, I was waiting for Otso, guess Otso was waiting for me.
Still learning how GitHub works.

DG12 committed:
fix comment

DG12 commented:
Restore transform:uppercase to &lt;P>, added transform:none to needsUpgrade.
I was waiting for Otso, Otso was waiting for me!
I'm still learning how GitHub is intended to work.

ojousima commented:
Sorry for the delay. We'll look into ruu.vi (and associated issue with beacon ID missing on current Ruuvi FW) alongside with v1-backports release.

DG12 commented:
Have you decided not to use this?
